portal/cookbook-tasklist.md

```diff
@@ -0,0 +1,51 @@
+# Cookbook Author Checklist Template
+
+Project Pythia maintainers will provide this checklist to Cookbook authors to help get Cookbooks ready for publication.
+
+```md
+**I am your Cookbook advocate, and my GitHub handle is <my-username>. Please tag me in this issue with any problems getting your Cookbook published!**
+Once we've marked this entire checklist, [click here to open an issue on ProjectPythia/cookbook-gallery to publish your Cookbook!](https://github.com/ProjectPythia/cookbook-gallery/issues/new?assignees=ProjectPythia%2Feducation&labels=content%2Ccookbook-gallery-submission&projects=&template=update-cookbook-gallery.yaml&title=Update+Gallery+with+new+Cookbook)
+
+---
+
+- [ ] **Confirm you’ve followed the entire Project Pythia [Cookbook Guide](https://projectpythia.org/cookbook-guide.html)**.
+Take note especially of the **Develop your cookbook**, **Authorship and the CITATION.cff file**, **Gallery tags**, and **Generate a DOI** sections. **Save the step of making a “release” of your cookbook as the last step of this checklist.**
```

Comment:
I advocate leaving the entire DOI and release generation until the end of the review process. The "release" should signify some level of "finished" product IMO. I also think the complexity of linking to Zenodo may turn some folks off. The Advocate should help shepherd this through at the end of the review process.

Here's my specific suggestion:
```suggestion
Take note especially of the [Develop your cookbook](https://projectpythia.org/cookbook-guide.html#develop-your-cookbook), [Authorship and the CITATION.cff file](https://projectpythia.org/cookbook-guide.html#authorship-and-the-citation-cff-file), and [Gallery tags](https://projectpythia.org/cookbook-guide.html#gallery-tags) sections. **Save the [Generate a DOI](https://projectpythia.org/cookbook-guide.html#generate-a-doi) step as the last step of this checklist.**
```